doc/cug.tex

```diff
@@ -5852,8 +5922,33 @@ \subsection{Suite And Task Event Handling}
     \item `retry' - the task failed but will retry
     \item `execution timeout' - task execution timed out
 \end{myitemize}
-MESSAGE, if provided, describes what has happened, and TASKID identifies
-the task (\lstinline=NAME.CYCLE= for cycling tasks).
+
+The value of each setting should be a list of shell scripts or shell script
+templates (see below).
```

Comment:
I'll settle for `command lines` and `command line templates` (as they are invoked with `shell=True`).